CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.0.1 (September 19, 2019)
```

Comment:
I think you introduced changes that are incompatible with the previous version. This should be 2.0.0. https://thoughtbot.com/blog/maintaining-open-source-projects-versioning